utils/to-doc.js

```diff
@@ -1,12 +1,16 @@
 'use strict'
 
 var extend = require('pouchdb-extend')
+var now = require('./now')
 
 module.exports = function objectToDoc (object) {
   var doc = extend({}, object, {
     _id: object.id
   })
 
+  doc.updatedAt = now()
+  doc.createdAt = doc.createdAt || doc.updatedAt
```

Comment:
I figured this transform was taking an object and creating a 'hoodie-style doc' so timestamps are typical of that, I'd be okay with an explicit method but not sure it's needed.